src/scripts/main.js

```diff
@@ -1,3 +1,187 @@
+
+  if (typeNotification === 'error') {
+    notification.classList.add('notification', 'error');
+    titleNotification.innerText = 'Incorrect data entered.';
+
+    if (info === 'age') {
+      textNotification.innerText = `Please check the ${info}. Minimum age 18, maximum 90.`;
+    } else {
+      textNotification.innerText = `Please check the ${info} and make corrections.`;
+    }
+  } else if (typeNotification === 'success') {
+    notification.classList.add('notification', 'success');
+    titleNotification.innerText = 'Success!';
+    textNotification.innerText = 'Data added successfully';
+  }
+
+  notification.append(titleNotification, textNotification);
+  document.body.append(notification);
+
+  window.setTimeout(() => notification.remove(), 3000);
+}
+
+function validationSalary(value) {
+  const clearNum = value.replace(/\D/g, '');
+  const formatedValue = parseFloat(clearNum).toLocaleString('en-US');
+
+  return '$' + formatedValue;
+}
+
+mainTable.tHead.addEventListener('click', function (eventData) {
+  if (!eventData.target.tagName === 'TH') {
+    return false;
+  }
+
+  const sortedElement = sortForTable(
+    this.parentElement,
+    eventData.target.cellIndex,
+  );
+
+  if (Event.sortBy !== eventData.target.textContent) {
+    Event.sortBy = eventData.target.textContent;
+    this.nextElementSibling.append(...sortedElement);
+  } else if (Event.sortBy === eventData.target.textContent) {
+    this.nextElementSibling.append(...sortedElement.reverse());
+    Event.sortBy = null;
+  }
+});
+
+mainTable.tBodies[0].addEventListener('click', function (eventData) {
+  const row = eventData.target.closest('tr');
+  const activeRow = this.querySelector('tr.active');
+
+  if (activeRow && activeRow !== row) {
+    activeRow.classList.remove('active');
+  }
+
+  if (row) {
+    row.classList.toggle('active');
+  }
+});
+
+newForm.addEventListener('submit', (evetntData) => {
+  evetntData.preventDefault();
+
+  const inputName = evetntData.target.elements.name;
+  const inputPosition = evetntData.target.elements.position;
+  const inputOffice = evetntData.target.elements.office;
+  const inputAge = evetntData.target.elements.age;
+  const inputSalary = evetntData.target.elements.salary;
+  const testForTextInput = /^[A-Z][a-z]+-?\s?[A-Z]?[a-z]+$/;
+
+  if (!testForTextInput.test(inputName.value) || inputName.value.length < 4) {
+    createMessage('error', 'name');
+
+    return false;
+  }
+
+  if (inputAge.value < 18 || inputAge.value > 90) {
+    createMessage('error', 'age');
+
+    return false;
+  }
+
+  const newRow = mainTable.tBodies[0].insertRow();
+
+  newRow.insertCell().textContent = inputName.value;
+  newRow.insertCell().textContent = inputPosition.value;
+  newRow.insertCell().textContent = inputOffice.value;
+  newRow.insertCell().textContent = inputAge.value;
+  newRow.insertCell().textContent = validationSalary(inputSalary.value);
+
+  createMessage('success');
+  evetntData.target.reset();
```

Comment:
There is a typo in the parameter name `evetntData`. It should be `eventData` to maintain consistency and avoid confusion.